### ISSUE TITLE
Readme

### ISSUE THREAD

**Issue by silvandiepen** [chiel/omni#1]
Chielll... wheres your readme?

**Comment by chiel** [comment]
Haha hey man - it's eh, not here. I would probably not use this yet if I were you 📦 

I'll add one soon!

**Comment by silvandiepen** [comment]
Wasn't planning on it, just wanted to know what it did, since I saw you were still working on it. :)

**Comment by chiel** [comment]
Yeah, I started a rewrite that's written in react and is entirely based on plugins, which makes it very easy to update the core of the cms, while having your client-specific stuff be unaffected. :)
